app/controllers/concerns/exception_handler.rb

```diff
@@ -19,7 +19,18 @@ module ExceptionHandler
       json_response({ :message => e.message }, :forbidden)
     end
 
-    rescue_from Exceptions::UserError do |e|
+    rescue_from RBACApiClient::ApiError do |e|
+      case e.code
+      when 403
+        json_response({ :message => e.message }, :forbidden)
+      when 404
+        json_response({ :message => e.message }, :not_found)
+      else
+        json_response({ :message => e.message }, :bad_request)
+      end
+    end
+
+    rescue_from Exceptions::UserError, Exceptions::ApprovalError do |e|
```

Comment:
Should we rescue all `ApprovalError` or only selected ones?

Reply:
All `ApprovalError` raised so far are suitable to category into bad requests.